### DISCUSSION TITLE
出現"是你的益達" 有錯誤的字元出現，可以幫忙修改相容性嗎，謝謝

### DISCUSSION THREAD

**Discussion by gloryhui1** [xiaohucode/yidaRule#183] [Q&A]
我用altstore安裝時出現"是你的益達" 有無效的字元出現，我的手機設定是繁體中文，可以幫忙修改相容性嗎，謝謝

**Comment by gloryhui1** [comment]
> 我用altstore安裝時出現"是你的益達" 有無效的字元出現，我的手機設定是繁體中文，可以幫忙修改相容性嗎，謝謝
我查到的是由于IPA包的Bundle display name 包含非英文字符。
解决方式：
修改info.plist文件 的 Bundle display name的值为英文（不能为中文）
可以幫忙嗎，謝謝

**Comment by CAiNSEMBRACE** [comment]
解决不了, 修改为英文字符后还会出其他的问题, 改用Sideloadly罢
